### DISCUSSION TITLE
Renovate does update lockfileVersion while engines are set in package.json

### DISCUSSION THREAD

**Discussion by rbel-io** [containerbase/base#1390] [Q&A]
Hi,

We have multiple front-end repos with a variation of node and npm versions.
For a single project Renovate updates the lockfileVersion from 1 to 3.
The engines are set in the package.json, this was the way to force installing this specific version..

```
  "config": {
    "engines": {
      "node": "14",
      "npm": "6"
    }
  },
```
This worked for all the projects that are currently on-boarded with again variation of node versions. Any idea what could cause this issue?

**Comment by viceice** [comment]
Please ask in main renovate repo if you don't know for sure it's a containerbase issue
